styles/index.css

```diff
@@ -0,0 +1,390 @@
+/* general */
```

Comment:
- [ ] There is a huge horizontal scroll on your mobile size as also requested to change by the previous reviewer. Kindly take a look at the below screenshot of your app and fix it 👇 
![Screenshot (254)_LI](https://user-images.githubusercontent.com/67087664/164892346-98d413fd-5226-4281-94da-71dc71d55136.jpg)

Reply:
@elyor-doniyorov  Pls do u have any suggestion regarding that as I'm already using flex can't code what the problem is

Reply:
@amklass11 I think the issue is coming from the partners section which is not mobile size.